modules/auxiliary/admin/kerberos/forge_ticket.rb

```diff
@@ -55,7 +56,21 @@ def initialize(info = {})
   SECS_IN_DAY = 60 * 60 * 24
 
   def run
-    enc_key = [datastore['NTHASH']].pack('H*')
+    validate_options
+
+    if datastore['NTHASH']
+      enc_type = Rex::Proto::Kerberos::Crypto::Encryption::RC4_HMAC
+      key = datastore['NTHASH']
+    else
+      key = datastore['AES_KEY']
+      if datastore['AES_KEY'].size == 64
+        enc_type = Rex::Proto::Kerberos::Crypto::Encryption::AES256
+      else
```

Comment:
Should we check the aes key size here too, and add an else clause with an error to the user that the key length was incorrect? 👀